fastddsspy_participants/src/cpp/visualization/ModelParser.cpp

```diff
@@ -214,65 +214,219 @@ ComplexParticipantData ModelParser::participants(const SpyModel& model, const dd
 std::vector<SimpleEndpointData> ModelParser::writers(const SpyModel& model)
 {
     std::vector<SimpleEndpointData> result;
-    // TODO
+
+    for (const auto& it : model.endpoint_database_)
```

Comment:
In general: could functions for readers and writers be reused? Use local auxiliary functions in this .cpp to encapsulate the repeated code. This is more important for verbose or specific methods, as they are more complex and have more code to maintain